resources/test-data.md

```diff
@@ -7,6 +7,17 @@ description: |
 menu_order: 1000
 ---
 
+<script language="javascript">
```

Comment:
Perhaps we should add this script to the theme instead, so we don't have it just on the `test-data.md` page? I'm sure @trieutran91 can assist with that.

Reply:
As I commented, I know it's not currently in the right place, haha